bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/PJLinkDeviceHandler.java

```diff
@@ -194,6 +249,25 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     }
   }
 
+  private @Nullable String getChannelTypeId(ChannelUID channelUID) throws ConfigurationException {
```

Comment:
ConfigurationException is never thrown by this method, is it ?

Reply:
True, I have removed the throws declaration now